extensions/wikia/FacebookPreferences/modules/ext.wikia.facebookPreferences.css

```diff
@@ -0,0 +1,9 @@
+.wds-button.facebook-button, .wds-button.facebook-button:hover {
+    background: #4b6ea8;
```

Comment:
hmmm, this is supposed to be same color as the Mercury Facebook button, but I'll re-check